CMakeLists.txt

```diff
@@ -41,18 +41,15 @@ string(REPLACE ";" "${AWS_MODULE_DIR};" AWS_MODULE_PATH "${CMAKE_PREFIX_PATH}${A
 # Append that generated list to the module search path
 list(APPEND CMAKE_MODULE_PATH ${AWS_MODULE_PATH})
 
-include(CMakePackageConfigHelpers)
-write_basic_package_version_file(
-        "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config-version.cmake"
-        COMPATIBILITY ExactVersion
-)
-
 if (NOT CMAKE_BUILD_TYPE)
     if (NOT WIN32)
         set(CMAKE_BUILD_TYPE "RelWithDebInfo")
     endif()
 endif()
 
+# Tell the CRT the version of the SDK for the metrics string
+set(AWS_IOT_SDK_VERSION ${SIMPLE_VERSION})
```

Comment:
compare this to what is used to be.
shouldn't it be something like `"v${FULL_VERSION}"` ?
so that it's prefixed with "v"
and has a bit of extra junk at the end for dev builds?

Reply:
Fixed! You were right, the metric string was incorrect for dev builds and needed `v`.

After several iterations, I got it generating the same way every time:
* Desired output (current CRT output): `v1.19.1-11-gfe4ab8e`
* Output without most recent changes: `1.19.1`
* Full version without changes: `v1.19.1-dev+fe4ab8e0`
* Simple version + Git hash (trimmed to 8 characters): `v1.19.1-fe4ab8e0`
  * While this solution provides the correct output, I realized that on a tagged release it will still include the git hash, which we do not want.
* Final solution (Full version with string replace): `v1.19.1-fe4ab8e0`

I also confirmed it gives the proper output if the `.git` folder is missing with the final solution.